src/main.rs

```diff
@@ -338,3 +360,21 @@ async fn css() -> impl Responder {
         .insert_header(ContentType(mime::TEXT_CSS))
         .message_body(css.into())
 }
+
+// Prints the given QrCode object to the console.
+fn print_qr(qr: &QrCode) {
+    let border: i32 = 4;
+    for y in (-border..qr.size() + border).step_by(2) {
```

Comment:
Can you print a two character-wide white border around these?

Reply:
I'm not sure I understand this properly~ I mean the content is already white. If doesn't scan then the problem more probably is with the rendering code `print_qr` because I modified it from https://github.com/nayuki/QR-Code-generator/blob/master/rust/examples/qrcodegen-demo.rs to show smaller code. But I have tested it locally and it scans well.

Can you try another scanner to pinpoint the cause here?

Reply:
I tested it and it works on my CLI if the colors are reversed (and with a border). If you want to try it:
`echo "lolomg" | qrencode -t UTF8` works but ``echo "lolomg" | qrencode -t UTF8i` doesn't.

Suggestion, since we have lots of horizontal space anyway: Just put the inversed and bordered version right next to the other one. This way, we'll make sure it works for everybody.